src/app/modal/modal.component.html

```diff
@@ -6,11 +6,9 @@
 >
   <div class="modal-content">
     <ng-content select="mz-modal-header"></ng-content>
-    <p>
-      <ng-content select="mz-modal-content"></ng-content>
-    </p>
+    <ng-content select="mz-modal-content"></ng-content>
```

Comment:
Like you said in the issue, the `<p>` element should be replaced with a `<div>`. Otherwise if somebody would write ...

```html
<mz-modal>
	<mz-modal-header>Header</mz-modal-header>
	<mz-modal-content>Content</mz-modal-content>
</mz-modal>
```

The content of `mz-modal-header` and `mz-modal-content` would be side-by-side instead of one over each other